gns3fy/gns3fy.py

```diff
@@ -1758,17 +1793,11 @@ def create_link(self, node_a, port_a, node_b, port_b):
                     node_id=_node_a.node_id,
                     adapter_number=_port_a["adapter_number"],
                     port_number=_port_a["port_number"],
-                    label=dict(
-                        text=_port_a["name"],
-                    ),
                 ),
```

Comment:
Is there a particular reason the label needs to be deleted?

Reply:
Regarding this, this change was done by mistake (actually I don't know how) and I had no intention to change this code.